cl/recap/tests.py

```diff
@@ -2063,9 +2064,9 @@ def test_orphan_documents_are_added(self, mock) -> None:
             upload_type=UPLOAD_TYPE.PDF,
             status=PROCESSING_STATUS.FAILED,
         )
-        process_recap_docket(self.pq.pk)
+        async_to_sync(process_recap_docket)(self.pq.pk)
         pq.refresh_from_db()
-        self.assertEqual(pq.status, PROCESSING_STATUS.SUCCESSFUL)
+        # self.assertEqual(pq.status, PROCESSING_STATUS.SUCCESSFUL)
```

Comment:
Why comment this out?

Reply:
Hmm, must have been left over from testing, uncommented now.